### DISCUSSION TITLE
{'error': 'HTTP 404 Not Found.  Please try again'}

### DISCUSSION THREAD

**Discussion by finalrange** [dpguthrie/yahooquery#225] [Q&A]
Hi all~i am facing an issue here. Some of the function , for instance: _ticker.key_stats_ and _ticker.price_ 
are showing me this error : {'error': 'HTTP 404 Not Found.  Please try again'}.

Anyone facing this issue? Any solution for this?

**Comment by yafm** [comment]
I get the same error trying to use the library. 

I notice a newer yahooquery module 2.3.3 but which now seems to need python3.9! i.e. I can't install the module on an older python.

Requirements in README.md still states "Python 2.7, 3.5+"

**Comment by goyahoo** [comment]
I have the same error too for the module : financial_data
Any solution esp. from the owner will be grateful.

**Reply by goyahoo** [reply]
I've just found the solution  by upgrading the yahooquery to its latest version.
